config/locales/en.yml

```diff
@@ -979,7 +979,7 @@ en:
             provider_type:
               blank: "Provider type can't be blank"
               format: Accredited provider cannot be a school
-            accrediting_provider:
+            accredited:
               format: '%{message}'
               invalid_provider_type: 'Accredited provider must not have Provider type school'
```

Comment:
Just a heads up, from some discussions with Simon, I am going to propose changing these provider types. Coupling the type and accreditation only leads to validation errors, we can infer the user-facing type based on accreditation and a lower-level provider type, such as "University" or "School" or "Institute".